lib/flip_card.dart

```diff
@@ -51,6 +51,8 @@ typedef void BoolCallback(bool isFront);
 class FlipCard extends StatefulWidget {
   final Widget front;
```

Comment:
These should be optional since `builder` would replace them in some cases.

Reply:
I hope explanation above is enough